src/time-picker/TimePicker.js

```diff
@@ -155,6 +156,8 @@ export const filterTime = ({ originalInput, minTime, maxTime }) => time => {
   return lteTime(minTime, time) && lteTime(time, maxTime) && startsWith(time, originalInput);
 };
 
+export const sortTimeList = timeList => sortBy(timeList, time => ( time.hours === 0 ? 24 : time.hours));
```

Comment:
how did you do that? I don't get this behaviour in my examples

![image](https://cloud.githubusercontent.com/assets/3502356/13610633/ec7d82f6-e55e-11e5-8734-99aab5916428.png)